404.html

```diff
@@ -6,9 +6,9 @@
   <script type="importmap">
     {
         "imports": {
-            "@dropins/tools/": "/scripts/__dropins__/tools/",
-            "@dropins/storefront-cart/": "/scripts/__dropins__/storefront-cart/",
-            "@dropins/storefront-auth/": "/scripts/__dropins__/storefront-auth/"
+          "@dropins/tools/": "/scripts/__/__dropins/tools/",
```

Comment:
In description you have this:

>Maintains complete package structure: scripts/__/@org/package-name/*

But the `__dropins` and `__adobe` that I've seen suggest otherwise. Was this because you cannot use `@` in the path?